netbench/netbench-driver/etc/Dockerfile

```diff
@@ -87,9 +93,14 @@ ENV RUST_BACKTRACE="1"
 
 # copy driver
 COPY --from=builder /app/bin /tmp/netbench
+ENV DEBIAN_FRONTEND=noninteractive
 RUN set -eux; \
+  apt update && apt install -y dnsutils curl unzip sudo; \
+  curl "https://awscli.amazonaws.com/awscli-exe-linux-aarch64.zip" -o "awscliv2.zip"; \
```

Comment:
Were you still planning on pulling CloudWatch logs for the server? If so, I would recommend a single method for uploading results and not upload as part of the container.

Reply:
If you're talking about moving logs from CloudWatch to S3, I created a separate lambda function for that which will be in my next PR.